src/components/export-modal/export-modal.spec.js

```diff
@@ -65,7 +66,10 @@ describe('ExportModal [Component]', () => {
     });
 
     it('renders the import checkbox', () => {
-      expect(component.find(`.${styles['export-to-lang-modal-checkbox']}`)).to.be.present();
```

Comment:
Is there any reason you were using styles instead of data-test-id?

Reply:
hmmm i can't remember! It was probably cause it couldn't find the item when I ran it. But if this is working, keep `data-test-id`